tf2_ros/include/tf2_ros/buffer_server.h

```diff
@@ -89,6 +92,14 @@ class BufferServer
       std::bind(&BufferServer::cancelCB, this, std::placeholders::_1),
       std::bind(&BufferServer::acceptedCB, this, std::placeholders::_1));
 
+    service_server_ =
+    rclcpp::create_service<LookupTransformService>(node->get_node_base_interface(), node->get_node_services_interface(), ns, std::bind(
+        &BufferServer::serviceCB, this, std::placeholders::_1,
+        std::placeholders::_2),
+        rmw_qos_profile_services_default,
+        nullptr
+        );
```

Comment:
Can someone explain to me why I get an error with:

```
    service_server_ = node->create_service<LookupTransformService>(ns, std::bind(
        &BufferServer::serviceCB, this, std::placeholders::_1,
        std::placeholders::_2));
```


I get: 
```
Starting >>> tf2_ros 
--- stderr: tf2_ros                             
In file included from /code/ros2_ws/src/geometry2/tf2_ros/src/buffer_server.cpp:41:
/code/ros2_ws/src/geometry2/tf2_ros/include/tf2_ros/buffer_server.h: In constructor ‘tf2_ros::BufferServer::BufferServer(const tf2::BufferCoreInterface&, NodePtr, const string&, tf2::Duration)’:
/code/ros2_ws/src/geometry2/tf2_ros/include/tf2_ros/buffer_server.h:95:65: error: expected primary-expression before ‘>’ token
   95 |     service_server_ =node->create_service<LookupTransformService>(ns, std::bind(
      |                                                                 ^
gmake[2]: *** [CMakeFiles/tf2_ros.dir/build.make:132: CMakeFiles/tf2_ros.dir/src/buffer_server.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:183: CMakeFiles/tf2_ros.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
---
Failed   <<< tf2_ros [2.48s, exited with code 2]
```

Reply:
I'm not sure.  But I'll point out that the timer has to do the same thing, so there is probably some template-reason that I don't understand.

Reply:
I don't understand it either, if there is no drawback to the current way I create the server then we can just leave it like that.

Reply:
> I don't understand it either, if there is no drawback to the current way I create the server then we can just leave it like that.

Yeah, there is no drawback as far as I understand.  So I'm fine with leaving it as-is.